server/datastore/mysql/hosts.go

```diff
@@ -4905,3 +4905,33 @@ func (ds *Datastore) GetHostHealth(ctx context.Context, id uint) (*fleet.HostHea
 
 	return &hh, nil
 }
+
+func (ds *Datastore) HostLiteByIdentifier(ctx context.Context, identifier string) (*fleet.HostLite, error) {
+	stmt := `
+    SELECT
+      h.id,
+	  h.team_id,
+      h.osquery_host_id,
+      h.node_key,
+      h.hostname,
+      h.uuid,
+      h.hardware_serial,
```

Comment:
Why do we need all these fields? For testing?

Reply:
I'm returning them because we are checking their fields (to load by identifier). Mostly for future usage.